### DISCUSSION TITLE
Tracing information is not displayed in the Table sometimes

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#9985] [Q&A] [answered]
![Table](https://user-images.githubusercontent.com/17882970/202602226-c81c507c-8e69-4508-afa8-72709fb5df94.gif)
Why the Tracing information is not displayed in the Table sometimes?

**Comment by Fine0830** [comment]
Please check whether there are errors on the `Console`.

**Reply by BertOnline** [reply]
@Fine0830 This is a screenshot of the console.

![VeryCapture_20221118105914](https://user-images.githubusercontent.com/17882970/202606949-cf03de0b-f5c5-4487-a78e-f780fce2ee9d.jpg)

**Reply by Fine0830** [reply]
I don't reproduce this issue. Can you provide your spans data?

**Reply by BertOnline** [reply]
@Fine0830 Can you reproduce this issue by my spans data?

**Reply by Fine0830** [reply]
Yes. Your data caused this problem. Check the `refs` in span data.

**Reply by BertOnline** [reply]
> Yes. Your data caused this problem. Check the `refs` in span data.

Is there a problem with the trace data generated or a bug in the UI? This is not the case in 8.9.1.

**Reply by wu-sheng** [reply]
https://github.com/apache/skywalking-booster-ui/pull/186

**Comment by Fine0830** [comment, marked as answer]
Fixed this via https://github.com/apache/skywalking-booster-ui/pull/186
